src/index.html

```diff
@@ -5,9 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <div class="card__item" data-qa="card">
+        <div class="card__top">
+          <div class="card__image">
+            <img
+              class="card__image-p"
+              src="./images/imac.jpeg"
+              alt="imac"
+            >
+          </div>
+          <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+          <div class="card__subtitle">Product code: 195434</div>
```

Comment:
use semantic tags for text elements
```suggestion
          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
          <p class="card__subtitle">Product code: 195434</p>
```